roles/create_rds_global_cluster/tasks/create.yml

```diff
@@ -0,0 +1,50 @@
+---
+- name: Run 'create_rds_global_cluster' role create operations
+  module_defaults:
+    group/aws: "{{ aws_setup_credentials__output }}"
+
+  block:
+    - name: Create rds global database
+      amazon.cloud.rds_global_cluster:
+        global_cluster_identifier: "{{ create_rds_global_cluster_global_cluster_name }}"
+        engine: "{{ create_rds_global_cluster_enigne  | default('aurora-postgresql')}}"
+        engine_version: "{{ create_rds_global_cluster_engine_version | default(omit) }}"
+        region: "{{ create_rds_global_cluster_primary_cluster_region }}"
+        state: present
+
+    - name: Create a primary cluster for global database "{{ create_rds_global_cluster_global_cluster_name }}" in "{{ create_rds_global_cluster_primary_cluster_region }}"
```

Comment:
Can you provide options to pass db_name , enable_global_write_forwarding, master_username, mater_user_password, vpc_security_group_ids ? These are essential when we want to utilize the cluster alongside other AWS resources.

Reply:
you may find [this](https://github.com/redhat-cop/cloud.aws_ops/pull/97/files#diff-4ab5ca0eb332969e1a5136f115d37608af1cc4a95e2e92e4436e5695e6c5fb1f) playbook helpful. It uses an Aurora cluster to set up a High Availability architecture

Reply:
thanks, will add these